pyproject.toml

```diff
@@ -15,6 +15,9 @@ PyYAML = ">=5.4.1"
 pyarrow = { version = ">=3.0.0", optional = true }
 etl-manager = { version = "^7.4", optional = true }
 pydbtools = { version = "^4.0.1", optional = true }
+psycopg2 = "^2.9.2"
+"testing.postgresql" = "^1.3.0"
+SQLAlchemy = "^1.4.27"
```

Comment:
I think these 3 need to be set to optional and `testing.postgresql` and `SQLAlchemy` included in the list for `postgres` dependencies (on line 33)

Reply:
Done